content/applications/productivity/calendar.rst

```diff
@@ -1,9 +1,285 @@
+:menuselection:`Calendar app --> Configuration --> Settings`. From here, enter
+:guilabel:`Client ID` and :guilabel:`Client Secret`. There is also an option to pause
+synchronization by ticking the checkbox, or automating synchronization by keeping it blank.
+
+Once the desired configurations are complete, be sure to click :guilabel:`Save` before moving on.
+
+Events created in synced calendars automatically appear across the integrated platforms.
+
+.. seealso::
+   - :doc:`Synchronize Outlook calendar with Odoo <calendar/outlook>`
+   - :doc:`Synchronize Google calendar with Odoo <calendar/google>`
+
+Create activities from chatter
+------------------------------
+
+Instantly create new meetings anywhere in Odoo through an individual record's chatter, like
+in a **CRM** opportunity card or task in the **Projects** app.
+
+From the chatter, click on the :guilabel:`Activities` button. In the :guilabel:`Schedule Activity`
+pop-up window, select the desired :guilabel:`Activity Type`, which populates a set of buttons,
+depending on the activity.
+
+Activities that involve other schedules, like :guilabel:`Meeting` or :guilabel:`Call for Demo`, link
+to the **Calendar** app. Select one of these activities to link to the **Calendar** app, then hit
+:guilabel:`Open Calendar` to navigate back to the app. Alternatively, it is also possible to
+:guilabel:`Schedule & Mark as Done` to close out the activity, or select :guilabel:`Done & Schedule
+Next` to keep the :guilabel:`Schedule Activity` window open to create another.
+
+.. seealso::
+  :doc:`Schedule activities in Odoo <../essentials/activities>`
+
+Plan an event
+-------------
+
+To put an event on the calendar, open the :menuselection:`Calendar app`, and click into the target
+date. On the :guilabel:`New Event` pop-up window that appears, start by adding the event title.
+
+.. image:: calendar/calendar-schedule-event.png
+   :alt: Schedule an event window on Calendar app.
+
+The target date auto-populates in the :guilabel:`Start` field. This can be changed by clicking
+into the date section, and selecting a date from the calendar. For multi-day events, select the end
+date in the second field, then click :guilabel:`Apply`.
+
+Tick the :guilabel:`All Day` checkbox if there is no specific start or end time.
+
+For events with specific start and stop times, ensure the :guilabel:`All Day` checkbox is unticked
+to enable time selection. With the :guilabel:`All Day` checkbox unticked, time selections appear in
+the :guilabel:`Start` field.
+
+The signed-in user auto-populates as the first attendee. Additional :guilabel:`Attendees` can be
+added or created from here, as well.
+
+For virtual meetings, copy and paste the URL into the space provided in the
+:guilabel:`Videocall URL` field. Or, click :icon:`fa-plus` :guilabel:`Odoo meeting` to create a
+link.
+
+Once the event is created, users can click into the virtual meeting directly from the calendar
+event.
+
```

Comment:
Also, this image (`calendar/calendar-new-meeting.png`) is unused. Seems like it would fit here, please include or remove :)
```suggestion

.. image:: calendar/calendar-new-meeting.png
   :alt: The full event form for a new calendar event.
   
```